### PR TITLE
Updated Flexible Mink to 2.0.x-dev

### PR DESCRIPTION
**Pull request by Chekote**
Updated project to use Flexible Mink 2.0.x so I can leverage the new Behat structure in stdcheck.

The project was outdated/broken in various ways, so I also had to fix a bunch of unrelated items. See commit messages for details.

### CODE REVIEW

**Comment by medology-bugman** [comment]
Testing file changes were not found.
Paging @PrinceMegahit to review the back-end work in this pull request!
Paging @RedEnchilada to review the testing work in this pull request!

I couldn't recognize who these files were relevant to:
.gitignore
composer.json
composer.lock

**Comment by Chekote** [comment]
This project seems broken (even on the master branch). I'm going to talk to Josh today about getting it fixed.

**Comment by Chekote** [comment]
This is ready for review now.

File: web/index.html [hunk 1]
```diff
@@ -95,7 +90,7 @@
             <input type="text" id="country" name="contact[country]" value="" style="display: none;" />
         </div>
     </div>
-    <script src="bower_components/jquery/dist/jquery.min.js"></script>
+    <script src="bower_components/jquery/dist/jquery.js"></script>
```
Comment (PrinceMegahit) [inline web/index.html]: Was this just for debugging?

File: web/index.html [hunk 2]
```diff
@@ -164,6 +159,6 @@ function fillInAddressImpl(){
             return self;
         }();
     </script>
-    <script src="https://maps.googleapis.com/maps/api/js?key=<?= getenv('GOOGLE_MAP_API_KEY'); ?>&amp;libraries=places&amp;callback=AutoComplete.init" async defer></script>
+    <script src="https://maps.googleapis.com/maps/api/js?key=YOUR_API_KEY&amp;libraries=places&amp;callback=AutoComplete.init" async defer></script>
```
Comment (PrinceMegahit) [inline web/index.html]: H-how does this work for CircleCI?

File: web/index.html (hunk 1 above)
Comment (Chekote) [reply web/index.html]: No. When we switched from Bower to Yarn, there is no longer a minified js. I believe the expectation is that you will use webpack or something to create a minified file of all your libraries. This file is just an example, so I didn't see much value in optimizing for js load times.

File: web/index.html (hunk 2 above)
Comment (Chekote) [reply web/index.html]: It doesn't. This project isn't setup to work on CircleCI.

When we do set this up, we'll just use an environmental variable and use sed to replace this.

**Pull request merged**